3rd-party/class.jetpack-amp-support.php

```diff
@@ -176,7 +176,7 @@ public static function amp_post_template_metadata( $metadata, $post ) {
 			$metadata = self::add_site_icon_to_metadata( $metadata );
 		}
 
-		if ( ! isset( $metadata['image'] ) ) {
+		if ( ! isset( $metadata['image'] ) && ! empty($post) ) {
```

Comment:
You'll want to use spaces inside the parenthesis, as per [the WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage):

```suggestion
		if ( ! isset( $metadata['image'] ) && ! empty( $post ) ) {
```

You should be able to click to accept the suggestion above, so you won't have to commit this manually.

Reply:
Oh interesting! Haha.